
led_config_list:
  
  - name: "front_right_leds"
    leds_zone: [1, 5]
    channel: 0
    type: "RGB"


  - name: "rear_right_leds"
    leds_zone: [6, 10]
    channel: 0
    type: "RGB"


  - name: "front_left_leds"
    leds_zone: [11, 15]
    channel: 0
    type: "RGB"


  - name: "rear_left_leds"
    leds_zone: [16, 20]
    channel: 0
    type: "RGB"


  - name: "all_leds"
    leds_zone: [1, 20]
    channel: 0
    type: "RGB"
